config/helm/chart/default/templates/Common/activegate/serviceaccount-activegate.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes, openshift, google" (include "dynatrace-operator.platformSet" .))}}
+{{ if eq (include "dynatrace-operator.partial" .) "false" }}
```

Comment:
`partial` is not really a meaningful name.
consider changing it to something like, e.g.,  `isCSI`

Reply:
my idea was that we could have other stuff in the future that we want to have a separate yaml for so that is why I chosen the name `partial`

if you look at the definition of `dynatrace-operator.partial` at https://github.com/Dynatrace/dynatrace-operator/pull/439/files#diff-212cec223aa4e2a1a063b33607deb1de9a46b96f698d7d9c0e57990f02a30cd7R100
it is `false` if not set to anything and `true` if set to anything.
In the future it could have multiple possible values like "csi" or "webhook" or whatever. So we only have 1 if per file for this instead of multiple.
 
I should replace this `partial=true` with `partial=csi` so it makes more sense (also do some other stuff)😅